src/screens/Settings/ChartTimeRange.js

```diff
@@ -0,0 +1,67 @@
+/* @flow */
+import React, { Component } from "react";
+import { View, FlatList } from "react-native";
+import { connect } from "react-redux";
+import SectionEntry from "../../components/SectionEntry";
+import LText from "../../components/LText";
+import { saveSettings } from "../../actions/settings";
+import type { SaveSettings } from "../../actions/settings";
+
+const mapDispatchToProps = {
+  saveSettings
+};
+
+const rangeList = {
```

Comment:
some remarks:
- instead of having to parseInt at some point, probably you should make rangeList an array of `{ name, days }`
- that array should be a prop of the component. and can just set this default in a defaultProps so it's transparent
- that array can be directly passed to the FlatList. you might need to change the keyExtractor to extract the `.days` or something like that. also probably can help a tiny bit to make keyExtractor being a static function & not recreate a reference in render()